recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,28 @@
+require inc/xt_shared_env.inc
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "CLOSED"
+
+GUEST = "${@bb.utils.contains('XT_GUESTS_INSTALL', 'doma', 'doma', '', d)}"
```

Comment:
we are using generic names for non-generic target: can we have GUEST_DOMA and UCONFIG_DOMA then?
Because `UBOOT_CONFIG[doma] = "${UCONFIG}" ` doesn't look generic here.
I mean , that when we add the same for DomD...

Reply:
yes, i think it is better do not introduce the UCONFIG at all and operate the UBOOT_CONFIG_prepend = "${GUEST} " to involve/not involve the domain into the processing.